beta/src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -13,13 +13,36 @@ import {
   SandpackReactDevTools,
 } from '@codesandbox/sandpack-react';
 import scrollIntoView from 'scroll-into-view-if-needed';
+import {useTimer} from 'use-timer';
 
 import cn from 'classnames';
 
 import {IconChevron} from 'components/Icon/IconChevron';
 import {NavigationBar} from './NavigationBar';
 import {Preview} from './Preview';
 import {CustomTheme} from './Themes';
+import {ReturnValue} from 'use-timer/lib/types';
+
+/**
+ * Temp component: testing purposes only
+ */
+const Timer: React.FC<{timer: ReturnValue}> = ({timer}) => {
+  const {listen} = useSandpack();
+
+  React.useEffect(() => {
+    const unsub = listen((message) => {
+      if (message.type === 'start') {
+        timer.start();
+      } else if (message.type === 'done') {
+        timer.pause();
+      }
+    });
+
+    return () => unsub();
+  }, []);
+
+  return <>{timer.time}ms</>;
```

Comment:
```suggestion
  return `${timer.time}ms`;
```

May be something like this?

Reply:
it doesn't matter, and also this is just a demo